packages/qdrant-loader/src/qdrant_loader/connectors/publicdocs/connector.py

```
@@ -306,16 +314,45 @@ async def get_documents(self) -> list[Document]:
+            )
+        if not self._initialized:
+            raise ConnectorNotInitializedError(
+                "Connector not initialized. Use async context manager."
+            )
+
+        if not self._should_process_url(entity_id):
+            return None
+        try:
+            content, title = await self._process_page(entity_id)
+        except (HTTPRequestError, DocumentProcessingError, ConnectorError) as e:
+            self.logger.error(f"Failed to fetch page {entity_id}: {e}")
```

Ruff (0.15.15)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)